typescript/agentkit/src/action-providers/vaultsfyi/schemas.ts

```diff
@@ -0,0 +1,55 @@
+import { z } from "zod";
+import { VAULTSFYI_SUPPORTED_CHAINS } from "./constants";
+
+/**
+ * Action schemas for the vaultsfyi action provider.
+ *
+ * This file contains the Zod schemas that define the shape and validation
+ * rules for action parameters in the vaultsfyi action provider.
+ */
+
+const networkSchema = z.enum(Object.values(VAULTSFYI_SUPPORTED_CHAINS) as [string, ...string[]]);
+
+/**
+ * Vaults list action schema.
+ */
+export const vaultsActionSchema = z.object({
```

Comment:
nit: We have not been camel casing the names. Our convention would make this `VaultsActionSchema`